src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,49 @@
+@import "../utils/variables";
+
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: $clock-dim;
+  height: $clock-dim;
+  border: 5px dotted black;
```

Comment:
Use HEX instead of color names. Color names can vary in rendering between browsers and devices. Hexadecimal values are more consistent and predictable across different platforms, ensuring that your chosen color appears as intended.

Reply:
Check other similar cases.

Reply:
Thanks! :)